Libraries/CMSIS/Device/Maxim/MAX32675/Include/max32675.svd

```diff
@@ -2,7 +2,7 @@
 <device xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaVersion="1.1" xsi:noNamespaceSchemaLocation="svd_schema.xsd">
  <vendor>Maxim-Integrated</vendor>
  <vendorID>Maxim</vendorID>
- <name>MAX32675</name>
+ <name>max32675</name>
  <series>ARMCM4</series>
```

Comment:
Is the lower case part name consistent with our other parts?

Reply:
Yes, I've been making this lowercase as I go through all our parts to pass ARM's SVDConv checks.